Jenkinsfile

```diff
@@ -1,11 +1,12 @@
 #!/usr/bin/env groovy
+@Library('pipeline-library@pull/465/head') _
```

Comment:
Will be removed once https://github.com/jenkins-infra/pipeline-library/pull/465 is merged.